### PR TITLE
LPS-109614 | master

### PR DESCRIPTION
**Pull request by Tim-Cao**
https://issues.liferay.com/browse/LPS-109614

### CODE REVIEW

**Comment by Tim-Cao** [comment]
ci:test:sf

**Comment by Tim-Cao** [comment]
ci:test:relevant

**Comment by manoelcyreno** [comment]
Just started reviewing :)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUI.testcase [hunk 1]
```diff
@@ -50,6 +50,73 @@ definition {
 			vocabularyName = "Vocabulary Name");
 	}
 
+	@description = "This is a use case for LPS-107730. Can not store 256 character in name field, key field and value field of category."
+	@priority = "4"
+	test CannotStore256CharactersInNameFieldKeyFieldAndValueField {
+		var actualText = "The number of characters is 256.The number of characters is 256.The number of characters is 256.The number of characters is 256.The number of characters is 256.The number of characters is 256.The number of characters is 256.The number of characters is 256";
+		var inputText = "The number of characters is 256.The number of characters is 256.The number of characters is 256.The number of characters is 256.The number of characters is 256.The number of characters is 256.The number of characters is 256.The number of characters is 256.";
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUI.testcase]: It has 257 characters, is it right?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUI.testcase [hunk 2]
```diff
@@ -50,6 +50,73 @@ definition {
...
+		var modifiedText = "The number of characters is 255.The number of characters is 255.The number of characters is 255.The number of characters is 255.The number of characters is 255.The number of characters is 255.The number of characters is 255.The number of characters is 255";
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUI.testcase]: It has 256 characters, is it right?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUI.testcase [hunk 3]
```diff
@@ -74,4 +141,140 @@ definition {
 			vocabularyName = "Vocabulary Name");
 	}
 
+	@description = "This is a use case for LPS-107730. Can input 254 characters to name field, key field and value field of category."
+	@priority = "4"
+	test Input254CharactersIntoNameFieldKeyFieldAndValueField {
+		var inputText = "The number of characters is 254.The number of characters is 254.The number of characters is 254.The number of characters is 254.The number of characters is 254.The number of characters is 254.The number of characters is 254.The number of characters is 25";
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUI.testcase]: It has 255 characters, is it right?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUI.testcase [hunk 4]
```diff
@@ -74,4 +141,140 @@ definition {
...
+			AssertTextEquals(
+				locator1 = "CategoriesEditCategory#PROPERTIES_VALUE_FIELD_1",
+				value1 = "${inputText}");
+		}
+	}
+
+	@description = "This is a use case for LPS-107730. Can input 255 characters to name field, key field and value field of category."
+	@priority = "5"
+	test Input255CharactersIntoNameFieldKeyFieldAndValueField {
+		var inputText = "The number of characters is 255.The number of characters is 255.The number of characters is 255.The number of characters is 255.The number of characters is 255.The number of characters is 255.The number of characters is 255.The number of characters is 255";
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUI.testcase]: It has 256 characters, is it right?

**Comment by manoelcyreno** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/85662.

**Comment by manoelcyreno** [comment]
ci:test:relevant

**Comment by manoelcyreno** [comment]
ci:test:sf

**Comment by manoelcyreno** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUI.testcase (hunk 1 above)
Comment (Tim-Cao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUI.testcase]: Nope. The value of inputText has 256 characters. I verified.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUI.testcase (hunk 2 above)
Comment (Tim-Cao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUI.testcase]: The value of modified text has 255 characters.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUI.testcase (hunk 3 above)
Comment (Tim-Cao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUI.testcase]: The value of inputText has 254 characters.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUI.testcase (hunk 4 above)
Comment (Tim-Cao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUI.testcase]: The value of inputText has 255 characters.

**Comment by manoelcyreno** [comment]
Just started reviewing :)

**Comment by manoelcyreno** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/85711.

**Pull request closed without merging**